exporter/exporterhelper/exporterhelper.go

```diff
@@ -0,0 +1,15 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package exporterhelper // import "go.opentelemetry.io/collector/exporter/exporterhelper"
+import "go.opentelemetry.io/collector/exporter/internal"
+
+// Request represents a single request that can be sent to an external endpoint.
+// Experimental: All the following APIs are at the early stage of development and may change without backward compatibility
+// until https://github.com/open-telemetry/opentelemetry-collector/issues/8122 is resolved.
+type Request = internal.Request
+type RequestErrorHandler = internal.RequestErrorHandler
```

Comment:
Can we keep the original docstring?  This will not render correctly, it will just add docs to `Request` but not to `RequestErrorHandler`

Reply:
Makes sense. Thank you, Pablo.